.github/workflows/CD.yml

```diff
@@ -22,11 +22,26 @@ jobs:
         run: |
           python3 -m pip install --upgrade build
           python3 -m build
-      - name: Upload
+      - name: Upload PYPI
         if: steps.semantic-release.outputs.released == 'true'
         run: |
           python3 -m pip install twine
           python3 -m twine upload --repository pypi dist/* -u __token__ -p ${{ secrets.PYPI_TOKEN }}
+      - name: Setup NODE
+        uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
+          registry-url: "https://registry.npmjs.org"
+      - name: Upload NPM
+        if: steps.semantic-release.outputs.released == 'true'
+        run: |
+          npm ci
+          rm -f -- schemas.json
```

Comment:
It would be simpler if the script directly either override the current file or remove the old one before writing the new one.